README.md

```diff
@@ -89,11 +89,10 @@ The following options are available:
  -p, --api-port             The REST Api port [=8080].
      --repo-kind            Backend for main repo store (fs, sqlite) [=fs].
  -q, --storage-quota        The size of the total storage quota dedicated to the node [=8589934592].
- -t, --block-ttl            Default block timeout in seconds - 0 disables the ttl [=$DefaultBlockTtl].
-     --block-mi             Time interval in seconds - determines frequency of block maintenance cycle: how
-                            often blocks are checked for expiration and cleanup
-                            [=$DefaultBlockMaintenanceInterval].
-     --block-mn             Number of blocks to check every maintenance cycle [=1000].
+ -t, --default-ttl          Default dataset expiry in seconds [=$DefaultDefaultExpiry].
+     --maint-interval       Maintenance interval in seconds - determines frequency of maintenance cycle:
+                            how often datasets are checked for expiration and cleanup. Value 0 disables the
+                            maintenance [=$DefaultMaintenanceInterval].
```

Comment:
Don't seem like it's resolving the default here

Reply:
You're right. I will change it to the literals `86400` (24 hours) for ttl and `300` for maintenance interval (5 minutes).

Reply:
I think this used to work fine (in fact previous versions of codex do resolve this fine), however, I'm seeing the same in #700. Looks like Duration isn't properly stringified.